design/ceph/ceph-cluster-cleanup.md

```diff
@@ -0,0 +1,65 @@
+# Ceph cluster clean up policy
+
+## Use case
+
+As a rook user, I want to clean up the datadirhostpath after I intentionally uninstall ceph cluster , so that I can start a new cluster without having to do any manual clean up.
+
+## Background
+
+If the user deletes a cluster and starts a new cluster on the same hosts, the path used by dataDirHostPath must be deleted. Otherwise, stale keys and other config will remain from the previous cluster and the new mons will fail to start. As of now, the user has to manually delete the dataDirHostPath on each host. This implementation aims to automate this deletion.
+
+## Design Flow
+
+### Preconditions
```

Comment:
This section just looks like the normal flow. Should we remove the Preconditions header?

Reply:
I think this section would be more clear if it were separated into 1) the principles behind the design such as why it is important to have user confirmation of the deletion, 2) What the user needs to do to give that confirmation (update the cluster CR with the new settings), and 3) How the operator implements the cleanup after the CR is updated.